gtars/src/uniwig/mod.rs

```diff
@@ -183,6 +95,12 @@ pub fn run_uniwig(matches: &ArgMatches) {
         .get_one::<i32>("threads")
         .expect("requires integer value");
 
+    let score = matches.get_one::<bool>("score").unwrap_or_else(|| &false);
```

Comment:
Why does `&false` need to be a reference? Arn't booleans a type that implements `Copy`?

Reply:
Compiler complains. Documentation shows using integers without reference but I dont see a reason for booleans.
![image](https://github.com/user-attachments/assets/31297a5a-2a09-490c-875c-1e2005b8e832)